File: examples/java/src/web/api/model/Contract.java
package web.api.model;

import java.util.List;

public class Contract {
	public String id;
    public String type;
    public String awardId;

    public String dateSigned;

    public String description;

    public String status;
    
	public List<Amendment> amendments;

	public Value value;
	
	public Period period;
	
	
	public Period getPeriod() {
		return period;
	}

	public void setPeriod(Period period) {
		this.period = period;
	}

	public Value getValue() {
		return value;
	}

	public void setValue(Value value) {
		this.value = value;
	}

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getType() {
		return type;
	}

	public void setType(String type) {
		this.type = type;
	}

	public String getAwardId() {
		return awardId;
	}

	public void setAwardId(String awardId) {
		this.awardId = awardId;
	}

	public String getDateSigned() {
		return dateSigned;
	}

	public void setDateSigned(String dateSigned) {
		this.dateSigned = dateSigned;
	}

	public String getDescription() {
		return description;
	}

	public void setDescription(String description) {
		this.description = description;
	}

	public String getStatus() {
		return status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

	public List<Amendment> getAmendments() {
		return amendments;
	}

	public void setAmendments(List<Amendment> amendments) {
		this.amendments = amendments;
	}
	
	
}

File: examples/csharp/OCDSApi/Models/Contract.cs
using System.Collections.Generic;

namespace OCDSApi.Models
{
    public class Contract
    {
        public string id { get; set; }
        public string type { get; set; }
        public string awardId { get; set; }

        public string dateSigned { get; set; }

        public string description { get; set; }

        public string status { get; set; }
        public string procurementMethod { get; set; }
        public string procurementMethodDetails { get; set; }

        public Period period { get; set; }
        public Value value { get; set; }
        public List<Item> items { get; set; }
        public List<Amendment> amendments { get; set; }

    }
}